samples/coroutines/kotlin/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 val platform: String by project
 val armArch: String by project
 val iosSigningIdentity: String by project
+val buildTarget: String? by project
```

Comment:
Actually I'm also against `buildTarget`. A target implies for me a targeted platform in a multiplatform project (which we are).
So how about `buildType` like it's called on the kotlin side anyways?

Reply:
I think we should discuss of that, in godot-cpp it is target, which I don't like too...

Reply:
Ya `buildType` sounds good to me